snap/snapcraft.yaml

```diff
@@ -190,13 +190,6 @@ parts:
     source: build-scripts/
     override-build: ./build-component.sh etcd
 
-  traefik:
```

Comment:
What happens when you refresh from a node with traefik to a node with the new apiserver-proxy? What happens to the old daemon?

Reply:
During the snap refresh, the traefik service is stopped and removed, handled by snapd

```
Jul 25 11:48:01 u2 snapd[1543]: services.go:919: RemoveSnapServices - disabling snap.microk8s.daemon-traefik.service
```

Old snap:

```
# before upgrade
root@u2:~# ps -fea | grep traefik
root       16573       1  0 11:46 ?        00:00:00 /snap/microk8s/x1/bin/traefik --configfile /var/snap/microk8s/x1/args/traefik/traefik.yaml
```

```
# after upgrade
root@u2:~# ps -fea | grep apiserver-proxy
root       18968       1  0 11:48 ?        00:00:00 /snap/microk8s/x2/bin/cluster-agent apiserver-proxy --config /var/snap/microk8s/x2/args/apiserver-proxy-config --kubeconfig /var/snap/microk8s/x2/credentials/kubelet.config --refresh-interval 30s
root       19777     636  0 11:49 pts/1    00:00:00 grep --color=auto apiserver-proxy
```